### DISCUSSION TITLE
Areabricks not registered

### DISCUSSION THREAD

**Discussion by jcnicke** [pimcore/pimcore#12798] [Q&A / Support]
I am installing Pimcore 10.5 community on Windows 2019 - Apache-2.4.51, Php-8.1.0, MySQL-8.0.30. I can launch public and admin, no problem.  
I am getting "Areabrick xxxxxxxx is not registered File: xxxx/xxxx/xxxx" throughout the pages.  

Looking for advice on Areabrick registration.

Thanks!

**Comment by passioneight** [comment]
Works for me [as documented](https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Editables/Areablock/Bricks.html):

![image](https://user-images.githubusercontent.com/57256630/182142969-e821b258-bfbd-4885-9263-42ba3eca3cb1.png)

What's the files' locations?

**Comment by jcnicke** [comment]
I have the global location <bundlePath>/templates/areas/<brickid>.  My bundle does not have a "Resources" directory.

**Reply by jdreesen** [reply]
Does your bundle class implement `getPath()` as [described here](https://symfony.com/doc/current/bundles/best_practices.html#directory-structure), then?

**Comment by jcnicke** [comment]
I am not sure. I am new to this environment and installed the demo bundle, making no changes.  Here is an example of the error:
Areabrick hero-teaser is not registered File: C:\wamp64\www\pimcore\my-project3\vendor\pimcore\pimcore\lib\Extension\Document\Areabrick\AreabrickManager.php Line: 129

I followed the install instructions (I think).  Did I miss something?

**Reply by jdreesen** [reply]
So there's no own/custom code except the official one from GitHub? In that case, I'd say the problem is using Windows, which is not officially supported.

I suggest running it with WSL and docker in that case.

**Reply by jdreesen** [reply]
See also [here](https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/Fix_Performance_Issues_on_Windows.html) for some tips.

**Comment by Stefan-JustMe69-SQ** [comment]
So, it's not supported (on windows i.e git bash) to do a git clone/create project of the demo-project and being able to run pimcore with arebricks?
That's what I see.
I did as described:

```
COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/demo my-project
cd ./my-project
./vendor/bin/pimcore-install
```

on my windows-pc
did as described here: https://github.com/pimcore/demo#getting-started

No bricks registered, same error as described

So description is wrong in the demo-package - and docker is not optional?

I see - there is a bug in the backlog:
#12938
